server/cluster.go

```diff
@@ -1157,36 +1157,23 @@ func (c *Cluster) rehash(nodes []string) []string {
 	return ringKeys
 }
 
-// Iterates over sessions hosted on this node and for each session
+// invalidateProxySubs iterates over sessions proxied on this node and for each session
```

Comment:
It would be nice to be able to re-hook proxy topic to the new master. But that can be done later.

Reply:
Fair enough. Added a TODO.